samples/drivers/dac/sample.yaml

```diff
@@ -4,10 +4,11 @@ tests:
   sample.drivers.dac:
     tags: DAC
     platform_allow: |
-      arduino_zero frdm_k22f frdm_k64f nucleo_f091rc nucleo_g071rb nucleo_g431rb
+      arduino_zero frdm_k22f frdm_k64f nucleo_f091rc nucleo_g071rb nucleo_g474re
```

Comment:
Out of curiosity: Why is the `nucleo_g431rb` removed? To reduce number of test runs for same SOC family? Is there any advantage to test with `g474` instead of `ge431`?

Reply:
We had a nucleo_g474re board available for testing, so it was choosen to represent the stm32g4 soc serie. The nucleo_g431re is still a valid target. Restoring

Reply:
Test results:
```
tests/drivers/dac/dac_api/drivers.dac,arm,nucleo_g474re,passed,,na,3.625081777572632,0,0
tests/drivers/dac/dac_api/drivers.dac,arm,nucleo_wl55jc,passed,,na,6.49380350112915,0,0
tests/drivers/dac/dac_api/drivers.dac,arm,nucleo_h743zi,passed,,na,3.405778169631958,0,0
samples/drivers/dac/sample.drivers.dac,arm,nucleo_g474re,passed,,na,2.5104336738586426,0,0
samples/drivers/dac/sample.drivers.dac,arm,nucleo_wl55jc,passed,,na,4.01857590675354,0,0
samples/drivers/dac/sample.drivers.dac,arm,nucleo_h743zi,passed,,na,2.741598606109619,0,0
```